nextflow.config

```
@@ -1,7 +1,7 @@
 process {
-    executor='lsf'
-    queue="$SCXA_LSF_QUEUE"
-    clusterOptions="$SCXA_LSF_OPTIONS"
+    executor='slurm'
+    queue="$SCXA_SLURM_QUEUE"
```

```suggestion
    queue="$SCXA_HPC_QUEUE"
```

Just so can be used in future for another scheduler if changed